api/rest/config.go

```diff
@@ -147,6 +153,21 @@ type jsonConfig struct {
 	CORSExposedHeaders   []string `json:"cors_exposed_headers"`
 	CORSAllowCredentials bool     `json:"cors_allow_credentials"`
 	CORSMaxAge           string   `json:"cors_max_age"`
+	HTTPLogFile          string   `json:"http_log_file"`
```

Comment:
You have attached the HTTPLogFile option to the bottom next to some CORS options. It would fit better after or before Auth